### PR TITLE
CLOUDP-239217 replace build-all with docker buildx bake

### PR DESCRIPTION
**Pull request by fmenezes**
This automates the tag generation and make it work with manifests

### CODE REVIEW

**Review by gssbzn — APPROVED** [review]

File: tools/aliaser/cmd/aliaser/main.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+}
+*/
+
+type manifest struct {
+	Tag             string              `json:"tag"`
+	TagAliases      map[string][]string `json:"tag_aliases"`
+	MongodbVersions []string            `json:"mongodb_versions"`
+}
+
+func run(r io.Reader, w io.Writer) error {
```
Comment (jeroenvervaeke) [inline tools/aliaser/cmd/aliaser/main.go:48]: Could we add some unit tests for this?

Comment (jeroenvervaeke) [reply tools/aliaser/cmd/aliaser/main.go:48]: @fmenezes do you think would it be interesting to run them as a github action?

Comment (jeroenvervaeke) [reply tools/aliaser/cmd/aliaser/main.go:48]: Non blocking

**Review by jeroenvervaeke — DISMISSED** [review]

File: tools/aliaser/cmd/aliaser/main.go (hunk 1 above)
Comment (fmenezes) [reply tools/aliaser/cmd/aliaser/main.go:48]: I'm holding it off until we are ready to push, I think this build-all will have a PUSH option and we will then submit as a github action in a cron job

File: tools/aliaser/cmd/aliaser/main.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	buf, err := io.ReadAll(r)
```
[the remark this answers is not in the document]
Comment (fmenezes) [reply tools/aliaser/cmd/aliaser/main.go:49]: done

**Review by blva — DISMISSED** [review]

**Review by jeroenvervaeke — APPROVED** [review]

**Pull request merged**